FoxHound.Client/src/AddBlog.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect, ChangeEvent, FormEvent } from "react";
+import axios from "axios";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+import Container from "@material-ui/core/Container";
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
```

Comment:
Let's try and do import {x,  y, z} from @material-ui/core and not list them all out as separate imports. Ryan had mentioned this has gotten better for compilation times. Have you noticed any significant performance differences one way or another?

Reply:
I can make these change, I didn't notice any performance differences.